android/src/main/kotlin/com/gdelataillade/alarm/models/AlarmSettings.kt

```diff
@@ -21,7 +26,7 @@ data class AlarmSettings(
     val vibrate: Boolean,
     val warningNotificationOnKill: Boolean,
     val androidFullScreenIntent: Boolean,
-    val allowAlarmOverlap: Boolean = false
+    val allowAlarmOverlap: Boolean = false // Defaults to false for backward compatibility
```

Comment:
I also this this is a better default, I wouldn't expect to alarms to ring at the same time as a novice user.